spiketoolkit/validation/qualitymetrics.py

```diff
@@ -2,7 +2,8 @@
 import spiketoolkit as st
 
 
-def compute_unit_SNR(recording, sorting, unit_ids=None, save_as_property=True):
+def compute_unit_SNR(recording, sorting, unit_ids=None, save_as_property=True, mode='mad',
+                     seconds=10, max_num_waveforms=1000, filter=False, freq_min=300, freq_max=6000):
```

Comment:
with median 300 is enough.

Reply:
I think 300 or 1000 doesn't change much in terms of time. It's fast anyways :)